tools/dependencies/known-dependencies.txt

```diff
@@ -289,3 +289,14 @@ zeppelin-client-0.10.1.jar
 zeppelin-common-0.10.1.jar
 zjsonpatch-0.3.0.jar
 zookeeper-3.4.14.jar
+aliyun-java-sdk-core-4.5.10.jar
```

Comment:
Please update the LICENSE.

Reply:
Sure, will update `dolphinscheduler-dist/release-docs/LICENSE`

Reply:
Updated in the latest commit.